<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::tab-bar {
   border: 1px solid gray;
}
QTabBar::tab {
  background: gray;
  color: white;
  padding: 10px;
 }

 QTabBar::tab:selected {
  background: rgb(0, 0, 0);
 }
QTabWidget::pane { 
   border: none;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Dataset</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom-right-radius:50px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/coronavirus.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>101</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/heart-beat.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/pain.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="pain_checkbox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pain</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="health_checkbox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>280</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Health</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="sng_checkbox">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>280</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sng</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dataset_next_button">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>320</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color:rgb(224, 224, 224);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255,255,255,255);
border-radius:20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#000000;&quot;&gt;Please select the condition type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>141</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(160,160,160);
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>141</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(160,160,160);
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>120</y>
        <width>141</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(160,160,160);
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>141</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(160,160,160);
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>160</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/question.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="test_checkbox">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>280</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
     <zorder>label_15</zorder>
     <zorder>label_22</zorder>
     <zorder>label_16</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>pain_checkbox</zorder>
     <zorder>health_checkbox</zorder>
     <zorder>sng_checkbox</zorder>
     <zorder>dataset_next_button</zorder>
     <zorder>textBrowser_3</zorder>
     <zorder>label_23</zorder>
     <zorder>label_18</zorder>
     <zorder>label_3</zorder>
     <zorder>label_6</zorder>
     <zorder>test_checkbox</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Traning Algorithms</string>
     </attribute>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255,255,255,255);
border-radius:20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="setting_next_button">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>320</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(255, 0, 0);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>511</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Video record</string>
      </property>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>481</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(160, 160, 160);</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Type of Drug:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mouse name:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="typeofdrug">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>base</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pain</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>vehicle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>jmf3737</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pregablin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ph5.2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ph7.3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Asic3Ko</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="NN_split">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
       <widget class="QLabel" name="batch_size">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Save path:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="mousename_editor">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="savepath_editor">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="LoadButton">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>10</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(255,255, 255);
    border-radius: 5px;
    font: 12px;
    padding: 3px;
}</string>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QDateTimeEdit {
    background-color: white;
    border: 1px solid gray;
    padding: 2px;
    border-radius: 3px;
    font-size: 12px;
}</string>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MonthSection</enum>
        </property>
       </widget>
       <widget class="QLabel" name="NN_split_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Recording time:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="recordingtime">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>1 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20 min</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QCheckBox" name="calibration_checkbox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>481</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(160, 160, 160);</string>
       </property>
       <property name="text">
        <string>Calibration</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>*如果有做過不需再做</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>361</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="setting_check_button">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>270</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color:rgb(224,224,224);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>check</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
     <widget class="QLabel" name="choosed_data">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Condition</string>
      </property>
     </widget>
     <widget class="QLabel" name="select_dataset">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/heart-beat.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionhome"/>
   <addaction name="actionsetting"/>
   <addaction name="actioncalibration"/>
   <addaction name="actionrecord"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionhome">
   <property name="icon">
    <iconset>
     <normaloff>image/home.png</normaloff>image/home.png</iconset>
   </property>
   <property name="text">
    <string>home</string>
   </property>
  </action>
  <action name="actionsetting">
   <property name="icon">
    <iconset>
     <normaloff>image/setting.png</normaloff>image/setting.png</iconset>
   </property>
   <property name="text">
    <string>setting</string>
   </property>
  </action>
  <action name="actioncalibration">
   <property name="icon">
    <iconset>
     <normaloff>image/measurement.png</normaloff>image/measurement.png</iconset>
   </property>
   <property name="text">
    <string>calibration</string>
   </property>
  </action>
  <action name="actionrecord">
   <property name="icon">
    <iconset>
     <normaloff>image/video.png</normaloff>image/video.png</iconset>
   </property>
   <property name="text">
    <string>record</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
